socorro/external/es/super_search_fields.py

```diff
@@ -192,6 +192,26 @@ def get_connection(self):
         with self.context() as conn:
             return conn
 
+    def get_supersearch_status(self):
+        """Return list of indices, latest index, and mapping.
+
+        :returns: dict with keys "indices", "latest_index", "mapping"
+
+        """
+        conn = self.get_connection()
+        index_client = elasticsearch.client.IndicesClient(conn)
+        indices = self.context.get_indices()
+        latest_index = sorted(indices)[-1]
+        doctype = self.context.get_doctype()
+
+        mapping = index_client.get_mapping(index=latest_index)
+        properties = mapping[latest_index]["mappings"][doctype]["properties"]
+        return {
+            "indices": indices,
```

Comment:
I was thinking it might be nice to know the index names and number of documents in them. I think I'm going to implement that now.